hnn_core/network.py

```diff
@@ -124,9 +125,7 @@ def __init__(self, params, n_jobs=1):
         # Global number of external inputs ... automatic counting
         # makes more sense
         # p_unique represent ext inputs that are going to go to each cell
-        self.p_common, self.p_unique = create_pext(self.params,
-                                                   self.params['tstop'])
```

Comment:
most of what is in this function needs to go inside `Rhythmic` and `Evoked` for backwards compatibility. With the new objects, users should be able to either provide legacy `params` to pre-populate these objects, or set them manually.